src/platform/K32W/BLEManagerImpl.cpp

```diff
@@ -743,13 +770,14 @@ CHIP_ERROR BLEManagerImpl::ConfigureAdvertisingData(void)
     ble_err_t err;
     CHIP_ERROR chipErr;
     uint16_t discriminator;
+    uint16_t advInterval = 0;
     gapAdvertisingData_t adv                              = { 0 };
     gapAdStructure_t adv_data[BLEKW_ADV_MAX_NO]           = { 0 };
     gapAdStructure_t scan_rsp_data[BLEKW_SCAN_RSP_MAX_NO] = { 0 };
     uint8_t advPayload[BLEKW_MAX_ADV_DATA_LEN]            = { 0 };
     gapScanResponseData_t scanRsp                         = { 0 };
     gapAdvertisingParameters_t adv_params                 = { 0 };
-    uint8_t chipAdvDataFlags                              = (gLeGeneralDiscoverableMode_c | gBrEdrNotSupported_c);
+    uint8_t chipAdvDataFlags                              = (gLeLimitedDiscoverableMode_c | gBrEdrNotSupported_c);
```

Comment:
Limited discoverable mode is no longer required in the latest spec AFAIK.

Reply:
Hope that I am using the latest spec. version. Chapter 5.2.3.8.4:

"Commissionable CHIP devices SHALL use the GAP Limited Discoverable mode, sending connectable undirected advertising events."

Reply:
I just checked the latest and it contains the following :)
"Commissionable CHIP devices SHALL use the GAP General Discoverable mode, sending connectableundirected advertising events."
and
"Bit 0 (LE Limited Discoverable Mode) SHOULD be set to 0
Bit 1 (LE General Discoverable Mode) SHOULD be set to 1"

Reply:
Thanks, reverted this change.